source/Octopus.Client/Serialization/Serializer.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Text;
+using Newtonsoft.Json;
+using Octopus.Cli.Extensions;
+
+namespace Octopus.Client.Serialization
+{
+    public static class Serializer
```

Comment:
Looking for feedback on naming/structure here - this was a bit of a "hack until it works" thing

Reply:
I should really add some tests around this too...